HoloPipelines/jobs/jobs_controller.py

```diff
@@ -47,10 +47,13 @@ def check_job_request_validity(job_request: dict) -> Tuple[bool, str]:
     return True, ""
 
 
-def job_success_callback() -> None:
+def job_success_callback(result: Optional[any]) -> None:
```

Comment:
I think in the other files I used `Any` (capital A), which I believe also neds to be imported from `typing`. But I don't know about `any`; if this also works, may even be the nicer solution.

Reply:
oh ya that's a built-in method not a type. Fixed accordingly.